src/wayland/wayland_base.cpp

```diff
@@ -143,13 +143,23 @@ mw::Global::~Global()
     wl_global_destroy(global);
 }
 
-void mw::internal_error_processing_request(wl_client* client, char const* method_name)
+void mw::post_implementation_error(wl_client* client, char const* fmt, ...)
 {
+    va_list va;
+    va_start(va, fmt);
 #if (WAYLAND_VERSION_MAJOR > 1 || (WAYLAND_VERSION_MAJOR == 1 && WAYLAND_VERSION_MINOR > 16))
```

Comment:
Maybe not in this PR, but I think we can drop this backwards-compatibility hack. We no longer target 16.04/core16